CHANGELOG.md

```diff
@@ -3,24 +3,28 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+## [1.3.7] - 2019-04-02
+
 ### Added
 - Added `add_to_history` kwarg to UVH5.write_uvh5_part
+- `_healpix_interp_bilinear` as a new interpolation method in `UVBeam`
+- `freq_interpolation_kind` added as an attribute to `UVBeam`
+- `tol` added as keyword argument to `UVBeam._interp_freq` which allows for a fast return of `data_array` slice if nearest-neighbor frequencies are all within the distance tolerance.
+- `polarizations` added as keyword argument to `UVBeam` interpolation methods.
 - Support for a yaml settings file to collect and propagate metadata for CST beam files.
 
+### Changed
+- `UVBeam._interp_freq` returns both `interp_data` and `interp_bandpass`, instead of just the former.
+
 ### Fixed
 - Combining overlapping data along multiple axes (most common when reading in multiple files) no longer errors.
 
 ## [1.3.6] - 2019-02-15
 
 ### Added
-- `_healpix_interp_bilinear` as a new interpolation method in `UVBeam`
-- `freq_interpolation_kind` added as an attribute to `UVBeam`
-- `tol` added as keyword argument to `UVBeam._interp_freq` which allows for a fast return of `data_array` slice if nearest-neighbor frequencies are all within the distance tolerance.
-- `polarizations` added as keyword argument to `UVBeam` interpolation methods.
 - `keep_all_metadata` keyword for optionally discarding unused metadata when performing a select operation.
 
 ### Changed
-- `UVBeam._interp_freq` returns both `interp_data` and `interp_bandpass`, instead of just the former.
 - Extends `run_acceptability_check` for UVH5 metadata in `check_header` function.
 
 ### Fixed
```

Comment:
One thing I noticed: version 1.3.4 has 2 "Added" sections. We should merge those into a single one.

Reply:
good catch, thanks!